src/main/kotlin/app/revanced/cli/command/utility/UninstallCommand.kt

```diff
@@ -48,19 +49,35 @@ internal object UninstallCommand : Runnable {
                 }.uninstall(packageName)
             } catch (e: Exception) {
                 logger.severe(e.toString())
+                throw e
             }
 
             when (result) {
-                RootInstallerResult.FAILURE ->
+                RootInstallerResult.SUCCESS ->
```

Comment:
I am just noticing code duplication here and in the patch subcommand, possibly extracting into a common function makes sense.

Reply:
> I am just noticing code duplication here and in the **patch** subcommand,

Did you mean `InstallCommand`?